### DISCUSSION TITLE
Batch bat file convert

### DISCUSSION THREAD

**Discussion by jadmanxxx** [ImageMagick/ImageMagick#4966] [Digital Image Processing]
hallo how can i make this to all files in one folder ?
importent that no new file names generate, overwrite orginal image file.

convert.exe C:\01\x.jpg -bordercolor white -border 20x20 \ -swirl 680 -implode .3  -shave 20x20 C:\01\x.jpg

thanks

**Comment by fmw42** [comment]
You can do that with mogrify

`mogrify.exe -bordercolor white -border 20x20 -swirl 680 -implode .3 -shave 20x20 *.jpg`

That will overwrite your input. Be sure to save a copy or use the -path option in mogrify. See https://legacy.imagemagick.org/Usage/basics/#mogrify

**Reply by jadmanxxx** [reply]
the script create  "*.jpg~" temp files and downt remove it ... you know how to fix this?

**Comment by jadmanxxx** [comment]
works perfect thanks !!!
the same with compress of images ?
convert.exe C:\01.jpg -quality 75 C:\01.jpg

**Reply by fmw42** [reply]
I do not understand. What is the question?  Do you simply want to compress a jpg?  If so

`mogrify.exe -quality 75 *.jpg`

**Reply by jadmanxxx** [reply]
yes thanks
